saleor/product/migrations/0104_product_minimal_variant_price.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 2.2.1 on 2019-06-10 11:23
+
+import django_prices.models
+from django.db import migrations
+
+
+def populate_product_minimal_variant_price(apps, schema_editor):
+    Product = apps.get_model("product", "Product")
+    for product in Product.objects.iterator():
+        # Set the "minimal variant price" to the default product's price
+        product.minimal_variant_price = product.price
+        product.save(update_fields=["minimal_variant_price"])
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("product", "0103_schema_data_enterprise_grade_attributes")]
+
+    operations = [
+        migrations.AddField(
+            model_name="product",
+            name="minimal_variant_price",
+            field=django_prices.models.MoneyField(
+                currency="USD", decimal_places=2, max_digits=12, null=True
+            ),
+        ),
+        migrations.RunPython(
+            populate_product_minimal_variant_price,
+            reverse_code=migrations.RunPython.noop,
+        ),
+        migrations.AlterField(
+            model_name="product",
+            name="minimal_variant_price",
+            field=django_prices.models.MoneyField(
+                currency="USD", decimal_places=2, max_digits=12
```

Comment:
You need to take the currency from `settings.DEFAULT_CURRENCY

Reply:
Thanks, good catch!
These were generated by django and I wouldn't notice it myself.